doc/src/tutorials/arch/multi_mode_ble.rst

```diff
@@ -0,0 +1,303 @@
+         <!-- These are the physical delay inputs on a Stratix IV LUT but because VPR cannot do LUT rebalancing,
+                  we instead take the average of these numbers to get more stable results
+             82e-12
+             173e-12
+             261e-12
+             263e-12
+             398e-12
+             397e-12
+             -->
+         <delay_matrix type="max" in_port="lut6.in" out_port="lut6.out">
+           261e-12
+           261e-12
+           261e-12
+           261e-12
+           261e-12
+           261e-12
+         </delay_matrix>
+       </pb_type>
+       <!-- Define flip-flop -->
+       <pb_type name="ff" blif_model=".latch" num_pb="1" class="flipflop">
+         <input name="D" num_pins="1" port_class="D"/>
+         <output name="Q" num_pins="1" port_class="Q"/>
+         <clock name="clk" num_pins="1" port_class="clock"/>
+         <T_setup value="66e-12" port="ff.D" clock="clk"/>
+         <T_clock_to_Q max="124e-12" port="ff.Q" clock="clk"/>
+       </pb_type>
+       <interconnect>
+         <direct name="direct1" input="ble6.in" output="lut6[0:0].in"/>
+         <direct name="direct2" input="lut6.out" output="ff.D">
+           <!-- Advanced user option that tells CAD tool to find LUT+FF pairs in netlist -->
+           <pack_pattern name="ble6" in_port="lut6.out" out_port="ff.D"/>
+         </direct>
+         <direct name="direct3" input="ble6.clk" output="ff.clk"/>
+         <mux name="mux1" input="ff.Q lut6.out" output="ble6.out">
+           <!-- LUT to output is faster than FF to output on a Stratix IV -->
+           <delay_constant max="25e-12" in_port="lut6.out" out_port="ble6.out"/>
+           <delay_constant max="45e-12" in_port="ff.Q" out_port="ble6.out"/>
+         </mux>
+       </interconnect>
+     </pb_type>
+     <interconnect>
+       <!--direct name="direct1" input="fle.in" output="ble6.in"/-->
+       <direct name="direct2" input="ble6.out" output="fle.out[0:0]"/>
+       <direct name="direct3" input="fle.clk" output="ble6.clk"/>
+     </interconnect>
+   </mode>
+   <!-- 6-LUT mode definition end -->
+
+Full example can be found at `link
+<https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/master/vtr_flow/arch/timing/k6_frac_N10_40nm.xml>`_.
+
+Validation in packer
+--------------------
+After finishing the architecture description, the next step is to validate that VPR can map logics to each operating modes.
+Since VPR packer will exhaustively try each operating mode and finally map logics to one of it.
+As long as there is an operating mode that is feasible for mapping, VPR will complete packing without errors.
+However, this may shadow the problems for other operating modes.
+It is entirely possible that an operating mode is not defined correctly and is always dropped by VPR during packing.
+Therefore, it is necessary to validate the correctness of each operating mode.
+Here, we will use the syntax ``disable_packing`` which can efficiently reach the goal.
```

Comment:
Probably should clarify that this is being done temporarily to validate the correctness of the mode definitions.